setup.cfg

```diff
@@ -21,6 +21,7 @@ package_dir =
 install_requires =
     numpy
     equistore @ https://github.com/lab-cosmo/equistore/archive/6ca7fa3.zip
+    skcosmo
```

Comment:
This should not be in the `install_requires` (it is an optional dependency, like ase or chemfiles)